os/arch/xtensa/src/esp32/esp32_timerisr.c

```diff
@@ -169,7 +169,7 @@ static int esp32_timerisr(int irq, uint32_t *regs, FAR void *arg)
 		 */
 
 		diff = xtensa_getcount() - compare;
-	} while (diff < divisor);
+	} while (diff > divisor);
```

Comment:
how about diff==divisor case? it seems need to loop one more time

Reply:
ok, it is corner case, should be consider.tks